src/main/java/formflow/library/ScreenController.java

```diff
@@ -562,14 +561,21 @@ private String createFormActionString(String flow, String screen) {
   }
 
   private Map<String, Object> createModel(String flow, String screen, HttpSession httpSession, Submission submission) {
+    return createModel(flow, screen, httpSession, submission, null);
```

Comment:
I'll make a change to just inline this reference rather than have the private method call an identically named version of itself.

Reply:
Ah, I hadn't check to see that there was only one other call to `createModel`... makes sense to do what you did then.